src/client/Routes.js

```diff
@@ -0,0 +1,28 @@
+// import React from 'react';
+
+import HomePage from './pages/HomePage';
+import SinglePage from './pages/SinglePage';
+import PageNotFound from './pages/NotFoundPage';
+
+// import PageNotFound from './pages/PageNotFound';
```

Comment:
@vitkuz are you going to use PageNotFound?

Reply:
Done